doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,139 @@
+.. _tutorial_qemu:
+
+Qemu
+****
+
+In this tutorial, we will create our first cloud-init user data config and
+deploy it into a Qemu_ virtual machine. We'll be using Qemu for this tutorial
+which is a popular emulator for running virtual machines on Linux. Several
+popular virtual machine tools use Qemu, including Libvirt, LXD, and Vagrant.
+
+Install Qemu
+============
+
+.. code-block:: sh
+
+    $ sudo apt install qemu-system-x86
+
+See qemu's `install instructions <https://www.qemu.org/download/#linux>`_.
+
+Download a Cloud Image
+======================
+
+All commands should be executed from a single directory; some
+commands create files that are used by later commands. One might want to
+use a temporary directory for easy clean up afterwards.
+
+.. code-block:: sh
+
+    $ wget https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
+
+Cloud images have cloud-init pre-installed.
+
+
+Define our user data
+====================
+
+Create the following file ``user-data``.
+
+.. code-block:: sh
+
+    $ cat << EOF > user-data
+    #cloud-config
+    password: passw0rd
+    chpasswd:
+      expire: False
+
+    EOF
+
+Define our meta data
+====================
+
+Create the following file on your local filesystem at ``meta-data``.
+
+.. code-block:: sh
+
+    $ cat << EOF > meta-data
+    instance-id: id-007
+    local-hostname: jammy
+
+    EOF
+
+
+Define our vendor data
+======================
+
+Not necessary, but faster (retry wait time).
+
+.. code-block:: sh
+
+    $ touch vendor-data
+
+
+Start an ad hoc IMDS Server
+===========================
+
+.. code-block:: sh
+
+    $ python3 -m http.server --directory . &
+
+
+Launch a vm with our user data
+==============================
+
+Now that we have LXD setup and our user data defined, we can launch an
+instance with our user data:
+
+.. code-block:: sh
+
+    $ qemu-system-x86_64                                            \
+        -net nic                                                    \
+        -net user                                                   \
+        -machine accel=kvm,type=q35                                 \
```

Comment:
> do you strictly need q35 type?

No

> almost everywhere "as fast as it can"

As a tutorial I think compatibility with various realease/distros is more important than speed. Fallback to tcg makes sense here. +1 thanks